### ISSUE TITLE
Deprecate `(post|comment|term|site) url` in favor of `* list --field=url`

### ISSUE THREAD

**Issue by danielbachhuber** [wp-cli/wp-cli#2070]
From https://github.com/wp-cli/wp-cli/pull/1826#issuecomment-100254973
- [x] comment url
- [x] post url
- [x] site url
- [x] term url
- [ ] <del>user url</code>

However, not all of the Query classes support querying by `*__in` and persisting the order. In particular, `WP_Comment_Query` only added support in WP 4.1

**Comment by danielbachhuber** [comment]
Actually, `orderby=>comment__in` was only added in 4.4 https://core.trac.wordpress.org/ticket/33883

**Comment by danielbachhuber** [comment]
`orderby=>include` was added to `WP_User_Query` in 4.1 https://core.trac.wordpress.org/ticket/30064

**Comment by ernilambar** [comment]
```
However, not all of the Query classes support querying by *__in and persisting the order. In particular, WP_Comment_Query only added support in WP 4.1
```

Re term: In `get_terms()`, `include` as value for `orderby` was introduced in `4.1`.

**Comment by ernilambar** [comment]
Update: include and orderby has some bug. Fixed in 4.7 https://core.trac.wordpress.org/ticket/37904

**Comment by danielbachhuber** [comment]
As it turns out, `wp user url` hasn't ever existed.
